pkg/instance/restart.go

```diff
@@ -0,0 +1,66 @@
+// SPDX-FileCopyrightText: Copyright The Lima Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package instance
+
+import (
+	"context"
+	"errors"
+	"time"
+
+	"github.com/lima-vm/lima/pkg/store"
+)
+
+const launchHostAgentForeground = false
+
+func Restart(inst *store.Instance) error {
```

Comment:
```suggestion
func Restart(ctx context.Context, inst *store.Instance) error {
```

Same for other functions too